src/config/Config.ts

```diff
@@ -39,25 +41,25 @@ export class Config {
 
   // Convenience accessors
   public static defaultHomeserverUrl(): string | undefined {
-    const defaultServerConfig = Config.instance.config.default_server_config;
+    const defaultServerConfig = Config.get().default_server_config;
     if (!defaultServerConfig) {
       return undefined;
     }
```

Comment:
I still don't think this is necessary

Reply:
Oh yeah, I after all that I forgot to remove this bit